src/herbpy/action/pouring.py

```diff
@@ -0,0 +1,104 @@
+import openravepy, os, time, prpy, actions, math
+from prpy.action import ActionMethod
+from herbpy.tsr.cup import *
+from herbpy.tsr.pitcher import *
+
+def PourWater(robot, cup, pitcher):
+	"""
+	@param robot The robot performing the pouring
+	@param cup The cup to pour into
+	@param pitcher The pitcher to pour
+	"""	
+
+	manip = robot.left_arm
+    manip_cup = robot.right_arm
+
+	# Grasp the handle
+    success = actions.GraspHandle(robot, pitcher)
+
+    #Lift Pitcher
+    if success:
+        success = actions.MoveObject(robot, direction=[0, 0, 1], distance=0.06)
+    
+    manip_pose = robot.GetActiveManipulator().GetEndEffectorTransform()   
+    
+    manip_cup.SetActive()
+    robot.SetActiveManipulator(manip_cup)
+    
+    #Grasp cup
+    success_cup = actions.PushGraspCup(robot, cup)
+
+    pitcher_start = pitcher.GetTransform()
+    pitcher_aabb = pitcher.ComputeAABB()
+
+    if success_cup:
+        success_cup = actions.MoveObject(robot, direction=[0, 0, 1], distance=0.01)
+
+    manip.SetActive()
+    robot.SetActiveManipulator(manip)
+
+    if success_cup:
+        p_tsr, min_tilt, max_tilt = robot.tsrlibrary(pitcher, 'pour')
+        traj = manip.PlanToTSR(p_tsr, execute=False)
+        cspec = traj.GetConfigurationSpecification()
```

Comment:
I think you want to use `GetActiveConfigurationSpecification` here. You may also need to set `manip.GetArmIndices()` as active earlier in this function to make this work as expected.